app/assets/stylesheets/active_admin/print.scss

```diff
@@ -228,61 +155,47 @@ table {
   }
 }
 
-// Sidebars
-#filters_sidebar_section {
-  display: none;
-}
-
 // Forms
-form {
-  fieldset {
-    border-top: thick solid #ccc;
-    padding-top: 2em;
-    margin-bottom: 2em;
+fieldset {
+  border: none;
+  margin: 0 0 2em 0;
+  padding: 2em 0 0 0;
 
-    &:last-child {
-      border-bottom: none;
-    }
+  fieldset {
+    border: 1px solid black;
```

Comment:
Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.